File: 4630-593-1431-kids-with-the-greatest-number-of-candies/4630-593-1431-kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> list = new ArrayList<>();

        int max = -1;

        for(int candy : candies)
            if(candy>max) 
                max = candy;

        for(int i=0;i<candies.length; i++)
            list.add((candies[i]+extraCandies >= max) ? true : false);
        
        return list;
    }
}

File: 4630-593-1431-kids-with-the-greatest-number-of-candies/4630-593-1431-kids-with-the-greatest-number-of-candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    ans = [];

    for(let candy of candies){
        ans.push(candy + extraCandies >= Math.max(...candies) ? true : false)
    }
    return ans;

};